tracpro/templates/polls/fetch_runs.html

```diff
@@ -4,11 +4,22 @@
 {% block pre-form %}
 
   <p>
-    {% trans "This manual fetch is only needed if you want to retrieve data from flow runs that happened before a panel was selected." %}*
-    {% trans "Enter for how many days in the past to retrieve flow run data. For example, if you enter 2, TracPro will fetch data from the last 48 hours." %}
+    {% blocktrans %}
+    This manual fetch is only needed if you want <b>to retrieve data
+    from flow runs that happened before a panel was selected.</b>
+    {% endblocktrans%}
+    *<br />
```

Comment:
We probably need to snuggle up the end of the sentence, the close tag, and the asterisk without whitespace, so the browsers will show it adjacent to the end of the sentence.  (e.g. ```selected.</b>{% endblocktrans %}*```)

Similarly at the beginning of the footnote, any whitespace will let the browser separate things.